docs/Makefile

```diff
@@ -53,13 +61,23 @@ livehtml: | $(CONDA_ENV_PYTHON)
 		--ignore _build \
         	--ignore .git \
         	--ignore */4913 \
-		--host $(HOSTNAME)\
-		--port 8080 \
```

Comment:
We want to keep this part so that when it binds to a host you can specify a DNS name.